packages/template/typescript-webpack/src/TypeScriptWebpackTemplate.ts

```diff
@@ -26,6 +26,9 @@ class TypeScriptWebpackTemplate extends BaseTemplate {
                 html: './src/index.html',
                 js: './src/renderer.ts',
                 name: 'main_window',
+                preload: {
+                  js: 'preload.js',
```

Comment:
Thanks for opening a fix for this so quickly. I see this is the typescript webpack template so should it reference the preload script with the `ts` file extension instead? When you initialize a project, I see that you get both `preload.js` and `preload.ts`. I thought this was odd and wasn't sure which one would be used.
```suggestion
                  js: 'preload.ts',
```

Reply:
Yes indeed. Thought that I changed it to ts, but I can see that it said .js. Thanks for the edit. The changes should be like the should.